### PR TITLE
React boilerplate added Fix #54 

### LINKED ISSUE #54: Find a suitable React boilerplate
Should find an open source, licence compatible, React boilerplate to start developing the front end. Having other needed plugins like Redux, testing tools, etc configured is a plus.

### PR DESCRIPTION
**Pull request by PMArtz92**
#54

### CODE REVIEW

File: OpenDF-UI/package.json [hunk 1]
```diff
@@ -0,0 +1,295 @@
+{
+  "name": "react-boilerplate",
```
Comment (agentmilindu) [inline OpenDF-UI/package.json]: Let's change these details to fit our project.

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,5 @@ _test
 OpenDF-ear/nb-configuration.xml
 .vagrant
 /sleuthkit-develop/sleuthkit-develop/bindings/java/test/
-/sleuthkit-develop/bindings/java/test/
\ No newline at end of file
+/sleuthkit-develop/bindings/java/test/
+/OpenDF-UI/node_modules/
```
Comment (agentmilindu) [inline .gitignore]: Shall we keep this in a `.gitignore` file inside the `OpenDF-UI` directory?

**Comment by PMArtz92** [comment]
fix the changes requested.

File: OpenDF-UI/package.json [hunk 3]
```diff
@@ -0,0 +1,295 @@
+{
+  "name": "Scorelab-OpenDF",
```
Comment (agentmilindu) [inline OpenDF-UI/package.json]: > New packages must not have uppercase letters in the name.

Let's change this to `scorelab-opendf`?

**Pull request merged**